aliasing/evaluators.py

```diff
@@ -332,22 +332,23 @@ def get_gvar(self, address):
             self._cache['gvars'][address] = result['value']
         return self._cache['gvars'][address]
 
-    def get_svar(self, name):
+    def get_svar(self, name, default=None):
         """
         Retrieves and returns the value of a svar (server variable).
 
         :param str name: The name of the svar.
-        :return: The value of the svar, or None if it does not exist.
+        :param default: What to return if the name is not set.
+        :return: The value of the svar, or the default value if it does not exist.
         :rtype: str or None
         """
         if self.ctx.guild is None:
             return None
         if name not in self._cache['svars']:
             result = self.ctx.bot.mdb.svars.delegate.find_one({"owner": self.ctx.guild.id, "name": name})
             if result is None:
-                return None
+                return default
             self._cache['svars'][name] = result['value']
-        return self._cache['svars'][name]
+        return self._cache['svars'][name] if self._cache['svars'][name] is not None else default
```

Comment:
is this change necessary? if the svar is in cache, it's guaranteed to be set (since we catch if it doesn't exist and return the default without caching on L349)